### DISCUSSION TITLE
How i can create a interation

### DISCUSSION THREAD

**Discussion by AndreZila01** [DSharpPlus/DSharpPlus#1385] [Support]
hey,

i would like understand how i can create a Interaction in one function, like this.
I try read your documentation about interation but i not undestand how i can create interation in one simple function.
Example one user reaction on message of bot 👍 ... i would like bot write 👍 . Its possible?

Example the message of bot:
![image](https://user-images.githubusercontent.com/56931353/188243068-4307a88f-5e7f-4a84-af57-7f5c78361335.png)

**Comment by AndreZila01** [comment]
I would like to understand. I want to create an integration every day, for example at 8 am, automatically without any command.
But your code is always with commands and that's what I'm asking for help, ... an interaction without the help of commands

**Reply by samjesus8** [reply]
So in your example where u said "Example one user reaction on message of bot 👍 ... i would like bot write 👍 . Its possible?"

Im assuming u want to get the bot to send 👍 when a user reacts to a message the Bot sends
In that case you just use interactivity when the Bot sends a message and wait for a user to react to the message, of course this has a timeout that you set in the InteractivityConfiguration of the Bot

U pretty much wanna do something like this
```csharp
[Command("react-example")]
public async Task Example(CommandContext ctx)
{
   var interactivity = ctx.Client.GetInteractivity();
   var message = new DiscordEmbedBuilder
   {
      Title = "React to this embed"
   };

   var messageToReact = await ctx.Channel.SendMessageAsync(embed: message);
   //Use interactivity to wait for a specific reaction
   //If this reaction is a thumbs up emoji, then you send back a thumbs-up emoji, should be easy for you to figure out how to do
}
```

Even better, if you know how event handler's work, you can use those which are better than using interactivity to be honest
